gcp/api/server.py

```diff
@@ -345,6 +346,32 @@ def DetermineVersion(self, request, context: grpc.ServicerContext):
     res = determine_version(request.query, context).result()
     return res
 
+  @ndb_context
+  @trace_filter.log_trace
+  def ImportFindings(self, request, context: grpc.ServicerContext):
+    """Return a list of `ImportFinding` for a given source."""
+    source = request.source
+    # TODO(gongh@): add source check
+    if not source:
+      context.abort(grpc.StatusCode.INVALID_ARGUMENT, 'Source is required.')
+    if get_gcp_project() == _TEST_INSTANCE:
+      logging.info('Checking import finding for %s\n', source)
+
+    query = osv.ImportFinding.query(osv.ImportFinding.source == source)
+    import_findings: list[osv.ImportFinding] = query.fetch()
+    invalid_records = []
+    for finding in import_findings:
+      invalid_records.append(
+          importfinding_pb2.ImportFinding(
```

Comment:
nit: can we add a `to_proto` method on `models.ImportFinding` instead? This puts the logic in a place closer to the actual model and is more consistent with how we do convert Bug to Vulnerability.

Reply:
added